src/pages/organize/[orgId]/people/duplicates/index.tsx

```diff
@@ -21,18 +21,23 @@ export const getServerSideProps: GetServerSideProps = scaffold(async () => {
 const DuplicatesPage: PageWithLayout = () => {
   const onServer = useServerSide();
   const { orgId } = useNumericRouteParams();
-  const list = useDuplicates(orgId).data ?? [];
+  const duplicates = useDuplicates(orgId);
   const messages = useMessages(messageIds);
 
   if (onServer) {
     return null;
   }
 
-  const filteredList = list.filter((cluster) => !cluster.dismissed);
+  const filteredList = (duplicates.data ?? []).filter(
+    (cluster) => !cluster.dismissed
+  );
 
   return (
     <>
-      {filteredList.length === 0 && (
+      {duplicates.isLoading && (
+        <CircularProgress color="inherit" size="1.5rem" />
```

Comment:
I feel like this is missing a margin/padding around it. This is what it looks like when I throttle my connection to "3G" in order to slow down loading:

![image](https://github.com/user-attachments/assets/2b6efa0d-29c6-42a0-9efa-3e7bbdd92c47)

The next case (empty state) has a 2 unit margin. The last case (list) has a 1.5 unit padding. I'm thinking all three should probably be the same instead?

Also, maybe the circular loading indicator should be centered? Wrapping it in a box looking something like this should work:

```tsx
<Box display="flex" justifyContent="center" m={2}>
  <CircularProgress />
</Box>
```